inductiva/fluids/scenarios/wind_tunnel/wind_tunnel.py

```diff
@@ -86,13 +104,16 @@ def simulate(self,
         """Simulates the wind tunnel scenario synchronously.
 
         Args:
+            simulator: Simulators available [OpenFOAM].
             object_path: Path to object inserted in the wind tunnel.
-            simulator: Simulator to use for the simulation.
             output_dir: Path to the directory where the simulation output
                 is downloaded.
             simulation_time: Simulation time, in seconds.
-            write_interval: Interval between simulation outputs, in seconds.
+            output_time_step: Interval between simulation outputs, in seconds.
```

Comment:
Before you replaced the unit specification ", in seconds" by "(s)". Why not do it here too?